specification/managementgroups/resource-manager/Microsoft.Management/stable/2020-05-01/management.json

```diff
@@ -1252,6 +1244,14 @@
         "parent": {
           "title": "Parent",
           "$ref": "#/definitions/ParentGroupInfo"
+        },
+        "path": {
+          "description": "The path from the root to the current group.",
+          "x-nullable": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ManagementGroupPathElement"
+          }
```

Comment:
May I know the reason why we move this from another struct to here? This is causing a breaking change in SDKs

Reply:
This was one of the breaking changes for which we received an approval from the Breaking Changes Review committee. This is actually correcting a mistake in the existing 2020-05-01 swagger: the "path" struct is being returned by the API under the "details" field, not under the "properties". All customers using our REST API have already taken a dependency on it being this way, so we can't modify the code. And the customers using SDKs are probably already broken, since the API returns the response in a different format.